tools/clh-img-check.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+usage="$(basename "$0") [-h] [-f n] -- program to check CLH compatible image
```

Comment:
"[]" usually means the parameter is optional. But "-f" is mandatory. So  the square brackets should be removed.

"-w" parameter is missing.

Reply:
Don't use the shortening "CLH"

Reply:
Ditto for the commit message, use `Cloud Hypervisor`.